homeassistant/components/fixer/sensor.py

```diff
@@ -21,7 +21,7 @@
 ATTR_EXCHANGE_RATE = "Exchange rate"
 ATTR_TARGET = "Target currency"
 
-DEFAULT_BASE = "USD"
+DEFAULT_BASE = "EUR"
```

Comment:
At first glance this looks like a breaking change for existing users.

Reply:
Yes but anyhow this plugin is dead due to the outdated api key layer

Reply:
ohh looks like not but code was incorrect 
![image](https://github.com/home-assistant/core/assets/29581516/7d1cd87d-4a24-4e82-a0bf-70b116d411e1)